server/controllers/chartsController/index.js

```diff
@@ -91,18 +94,23 @@ export const graphDataController = async (
     })
   }
 
-  const dataBySite = postProcessData(dataMap, studyTotals)
+  const postProcessedDataBySite = postProcessData(dataMap, studyTotals)
 
   if (isAnyTargetIncluded(studyTotals)) {
     labelMap.set(N_A, { name: N_A, color: '#808080' })
   }
 
+  const dataBySite =
+    allSubjects.length > 0 ? Array.from(postProcessedDataBySite.values()) : []
+  const labels = Array.from(labelMap.values()) || []
```

Comment:
`Array.from(new Map())` is an empty array so I don't think we need to default this.

```suggestion
  const labels = Array.from(labelMap.values())
```

Reply:
881cc90